desktop/extensions-sdk/dev/continuous-integration.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Continuous Integration"
+description: Automatically test and validate your extension.
+keywords: Docker, Extensions, sdk, CI, test, regression
+---
+
+In order to help validating your extension and ensure it's functional, Docker Desktop provides a set of tools to help you set continuous integration for your extension.
```

Comment:
Maybe we should mention that they are experimental, as we did in the package readme. WDYT @gtardif?